README.md

```diff
@@ -24,7 +24,7 @@ This app requires a dedicated root domain. **ARM installations are NOT SUPPORTED
 
 ## Configuration
 
-The app's config file is stored in `/opt/yunohost/__APPNAME__/nitter.conf`. 
+The app's config file is stored in `/opt/yunohost/nitter/nitter.conf`. 
```

Comment:
If the user installs the app multiple times, the config file could be at `nitter__2`, `nitter__3`, etc

Reply:
It's maybe worth specifying that in the readME and give the config link of the first instance: `/opt/yunohost/nitter/nitter.conf`